### PR TITLE
Terraform - Add SNS, SQS, ffmpeg layer

### PR DESCRIPTION
**Pull request by shadman0711**
Fixes #

- [ ] Does not depend on other PRs
- [x] Depends on other PR being merged first: #191 

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [ ] New feature
- [ ] Refactor
- [x] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
Terraform configuration for SNS, SQS, Lambda layer added

## What was done
* SNS added
* SQS added
* FFMPEG Lambda Layer Added

## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: terraform/.terraform.tfstate.lock.info [hunk 1]
```diff
@@ -0,0 +1 @@
+{"ID":"1f44a41c-8cd3-869c-4472-e1140cbe8c4c","Operation":"OperationTypePlan","Info":"","Who":"craftsmen2@craftsmen-Latitude-E6530","Version":"1.2.7","Created":"2022-08-30T08:20:05.651233219Z","Path":"terraform.tfstate"}
```
Comment (mhazad) [inline terraform/.terraform.tfstate.lock.info:1]: This should not be part of the git

File: terraform/lambdas.tf [hunk 2]
```diff
@@ -5,9 +5,21 @@ resource "aws_lambda_function" "terraform_lambda" {
...
+  layers  = [aws_lambda_layer_version.lambda_layer.arn]
+  timeout = 600
+
+  environment {
+    variables = {
+      SNS_TOPIC_ARN       = "arn:aws:sns:${var.aws_region}:${data.aws_caller_identity.current.account_id}:${var.sns_topic_name}"
+      DYNAMODB_TABLE_NAME = var.ddb_table_name
+      S3_BUCKET_NAME      = var.s3_bucket_name
+      SNS_TOPIC_ARN       = aws_sns_topic.ddb_insert_record.arn
+      WEBHOOK_URL         = "https://discord.com/api/webhooks/1004259721424941076/Xo4Lntkm0yycgx1uXkAZ8rh6EonlWfw6Rb6ukBlGI4-sW4lkF5FwvfJnDzkmZtdwWV_d"
```
Comment (mhazad) [inline terraform/lambdas.tf:20]: These should be read from the environment.

File: terraform/sqs.tf [hunk 3]
```diff
@@ -0,0 +1,28 @@
+resource "aws_sqs_queue" "terraform_queue" {
+  name                       = var.sqs_queue_name
+  visibility_timeout_seconds = 600
+
+  # https://aws.amazon.com/premiumsupport/knowledge-center/sqs-s3-event-notification-sse/
```
Comment (mhazad) [inline terraform/sqs.tf:5]: What is the purpose of this commented out code?

**Review by mhazad — CHANGES_REQUESTED** [review]
Please address the feedback.

File: terraform/lambdas.tf (hunk 2 above)
Comment (shadman0711) [reply terraform/lambdas.tf]: Removed this file in commit [e5917b2](https://github.com/CraftsmenLtd/coe-batch-q322/pull/310/commits/e5917b2eec774b5eee6db22061fcb4e0f82fa936)

File: terraform/sqs.tf (hunk 3 above)
Comment (shadman0711) [reply terraform/sqs.tf]: Removed commented code in [e5917b2](https://github.com/CraftsmenLtd/coe-batch-q322/pull/310/commits/e5917b2eec774b5eee6db22061fcb4e0f82fa936)

File: terraform/.terraform.tfstate.lock.info (hunk 1 above)
Comment (shadman0711) [reply terraform/.terraform.tfstate.lock.info]: Removed this file in commit [e5917b2](https://github.com/CraftsmenLtd/coe-batch-q322/pull/310/commits/e5917b2eec774b5eee6db22061fcb4e0f82fa936)

File: terraform/lambdas.tf (hunk 2 above)
Comment (shadman0711) [reply terraform/lambdas.tf]: Added webhook url variable in commit [e5917b2](https://github.com/CraftsmenLtd/coe-batch-q322/pull/310/commits/e5917b2eec774b5eee6db22061fcb4e0f82fa936)

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**